lib/update.sh

```diff
@@ -0,0 +1,193 @@
+restname="FileFighterREST"
+frontendname="FileFighterFrontend"
+networkname="FileFighterNetwork"
+dbname="FileFighterDB"
+
+
+ffupdate(){
+
+
+
+if [[ $(docker ps -a --format "{{.Names}}" | grep $restname) ]] || [[ $(docker ps -a --format "{{.Names}}" | grep $frontendname) ]] || [[ $(docker ps -a --format "{{.Names}}" | grep $dbname) ]] || [[ $(docker ps -a --format "{{.Names}}" | grep $reverseproxyname) ]]; then
+   echo "Installation is fine, starting update"
+else
+  echo "FileFighter is not installed, run 'ffighter install' first"
+fi
+
+  source lib/config.sh     # load the config library functions
+  source lib/dockertags.sh # load docker functions.
+
+configFilePath=$(pwd)/config.cfg
+
+  # Read in default keys.
+  app_port="$(read $configFilePath app_port)"
+  db_name="$(read $configFilePath db_name)"
+  db_user="$(read $configFilePath db_user)"
+  db_password="$(read $configFilePath db_password)"
+  use_stable_versions="$(read $configFilePath use_stable_versions)"
+
+
+  if ! [[ $app_port ]]; then
+    echo "Config for app_port not found, using defaults."
+    app_port="$(read ./lib/config.cfg.defaults app_port)"
+  fi
+
+  if ! [[ $db_name ]]; then
+    echo "Config for db_name not found, using defaults."
+    db_name="$(read config.cfg.defaults db_name)"
+  fi
+
+  if ! [[ $db_user ]]; then
+    echo "Config for db_user not found, using defaults."
+    db_user="$(read config.cfg.defaults db_user)"
+  fi
+
+  if ! [[ $db_password ]]; then
+    echo "Config for db_password not found, using defaults."
+    db_password="$(read config.cfg.defaults db_password)"
+  fi
+
+  # Finished Config:
+  echo "Finished reading config. starting update..."
+
+  # Check if (default) password was empty.
+  if ! [[ $db_password ]]; then
+    echo "Password was not set, please install first."
```

Comment:
oks set the password in your config `configPath` or run 'ffigther install' again.